addons/bus/static/src/workers/websocket_worker.js

```diff
@@ -261,6 +290,15 @@ export class WebsocketWorker {
         this.isReconnecting = false;
     }
 
+    /**
+     * Try to reconnect to the server, an exponential back off is
+     * applied to the reconnect attempts.
+     */
+    _retryConnectionWithDelay() {
+        this.connectRetryDelay = this.connectRetryDelay * 1.5 + 500 * Math.random();
+        this.connectTimeout = setTimeout(this._start.bind(this), this.connectRetryDelay);
+    }
```

Comment:
Slightly out-of-scope of PR, but I'm wondering whether current code could not call this several times and call `_start` several times?